Maestro/subscriptions.json

```diff
@@ -823,6 +823,7 @@
         "https://github.com/aspnet/AspLabs/blob/release/**/*",
         "https://github.com/aspnet/AspNetCore/blob/master/**/*",
         "https://github.com/aspnet/AspNetCore/blob/release/**/*",
+        "https://github.com/aspnet/AspNetCore-Internal/blob/master/**/*",
```

Comment:
If there's any chance of release branches being useful, a proactive trigger here wouldn't break anything.